### ISSUE TITLE
BLOCKED: Add GP WOE places not already in WOF

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst/whosonfirst-data#109]

**Comment by nvkelso** [comment]
These would end up on Null Island since there aren't any geometries?

**Comment by thisisaaronland** [comment]
Names, adjacencies and hierarchies and in particular parent IDs for which we probably do have a concordance and can generate an approximate centroid. Also WOE was a project with active editorial work so unless it's a (WOE) POI type then I'd default to it being not complete crazy-talk.

**Comment by nvkelso** [comment]
For any approximate centroids we should mark them `"mz:is_approximate": 1`

- https://github.com/whosonfirst/whosonfirst-properties/blob/29f4c22e0c308b5227ba6e7ee333822a0b0bff80/properties/mz.md#is_approximate

**Comment by nvkelso** [comment]
Same placetype logic as the GeoNames import:

This would be any place in GeoNames that doesn't already have a concordance, in the following general place types:

- `country` (deprecate)
- `disputed` (deprecate)
- `dependency` (deprecate)
- `continent` (deprecate)
- `ocean` (deprecate)
- `marinearea` (deprecate)
- `macroregion` (deprecate)
- `region` (deprecate)
- `macrocounty` (deprecate)
- `county` (deprecate)
- `localadmin` (by country if Quattroshapes had good polygons deprecate e.g. in UK, France, else import)
- `locality`
- `neighbourhood` (with special handling for Zach's polygons and Nat's points)

**Comment by nvkelso** [comment]
This seems like it has limited utility now?

**Comment by stepps00** [comment]
Agreed. Concordance values would be useful to add, but since the GeoNames import imported millions of new locality records (and we've QA'd the other placetypes), I'd hesitate to add more, new records for administrative places. 

If there are source properties or concordances, though, I'm all for importing those.
